pages/account/manage/event/[[...data]].js

```diff
@@ -63,23 +58,51 @@ export default function ManageEvent({ BASE_URL, event }) {
   const [name, setName] = useState(event.name || "");
   const [description, setDescription] = useState(event.description || "");
   const [url, setUrl] = useState(event.url || "");
-  const [startDate, setStartDate] = useState(
-    event.date?.start && formatDate(event.date?.start)
-  );
-  const [endDate, setEndDate] = useState(
-    event.date?.end && formatDate(event.date?.end)
-  );
+  const [startDate, setStartDate] = useState("");
+  const [endDate, setEndDate] = useState("");
   const [price, setPrice] = useState(event.price?.startingFrom || 0);
   const [color, setColor] = useState(event.color || "");
 
+  const formatLocalDate = (inputDate) => {
+    const d = new Date(inputDate);
+    const year = d.getFullYear();
+    const month = ("0"+(d.getMonth() +1)).slice(-2);
+    const day = ("0"+d.getDate()).slice(-2)
+    const date = `${year}-${month}-${day}`;
+    const time = d.toTimeString().split(":");
+
+    return `${date}T${time[0]}:${time[1]}`;
+  };
```

Comment:
Should this be moved to the `services/utils/dateFormat.js`?

Reply:
This is useful only for `datetime-local` input fields.

If we expect to need more of those, I can move it to a util file.

Reply:
Oh I see, ok this can be moved later